### PR TITLE
Update @microsoft/decorators to build with a manifest

### PR DESCRIPTION
**Pull request by pgonzal**
This approach was suggested by @iclanton

### CODE REVIEW

File: libraries/decorators/gulpfile.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 'use strict';
 
-const build = require('@microsoft/node-library-build');
+const build = require('@microsoft/web-library-build');
```
Comment (iclanton) [inline libraries/decorators/gulpfile.js]: This should be `sp-build-web`.

File: libraries/decorators/karma.config.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
+'use strict';
```
Comment (iclanton) [inline libraries/decorators/karma.config.js]: You shouldn't need this file if you're using sp-build-web.

**Comment by qz2017** [comment]
> 

I am not convinced this is the right thing to do.

---
Refers to: libraries/decorators/src/decorators.manifest.json:10 in 151c757. [](commit_id = 151c757f3f0e71337f7944eedc4d0c6b19909a44, deletion_comment = False)

**Comment by qz2017** [comment]
:clock1:

**Comment by nickpape-msft** [comment]
Are you still trying to merge this @pgonzal ? Otherwise we should close it.

**Comment by pgonzal** [comment]
I will abandon this PR.  The reason we were trying to move **@microsoft/decorators** into **web-build-tools** was so that API Extractor could take a dependency on it.  However, the design has changed, and going forward the @virtual/@override/@sealed metadata will be represented using JSDoc (because decorators don't show up in the *.d.ts files).

I'll create a different PR to remove decorators from this repo.

---
In reply to: [375068476](https://github.com/Microsoft/web-build-tools/pull/250#issuecomment-375068476) [](ancestors = 375068476)

**Pull request closed without merging**